pkg/server/handler/tikvhandler/tikv_handler.go

```diff
@@ -1013,7 +1013,11 @@ func (h SchemaHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		// all table schemas in a specified database
 		if schema.SchemaExists(cDBName) {
 			if a := req.FormValue(handler.IDNameOnly); a == "true" {
-				tbs := schema.SchemaSimpleTableInfos(context.Background(), cDBName)
+				tbs, err := schema.SchemaSimpleTableInfos(context.Background(), cDBName)
```

Comment:
Please add a test case for it, just like https://github.com/pingcap/tidb/blob/f00bf9a977f68cde25b2be53a95c411693901bc9/pkg/server/handler/optimizor/optimize_trace_test.go#L1

Reply:
Why we use panic in the past? because it's difficult to mock such kind of error.
The only case until now is tikv unavailable.
@bb7133

Reply:
I add a basic test to cover the HTTP request, but not the return error path.